packages/account-v2/src/containers/ManualForm/manual-form-inputs.tsx

```diff
@@ -34,21 +33,19 @@ export const ManualFormInputs = ({ isExpiryDateRequired, selectedDocument }: TMa
                 </Field>
                 {isExpiryDateRequired && (
                     <Field name='document_expiry'>
-                        {({ field, form, meta }: FieldProps) => {
+                        {({ field, meta }: FieldProps) => {
                             const hasError = meta.touched && !!meta.error;
                             const fieldLabel = `${fieldsConfig.documentExpiry.label}*`;
                             return (
-                                <DatePicker
+                                <Input
                                     {...field}
                                     aria-label={fieldLabel}
                                     autoComplete='off'
                                     className='w-full'
-                                    errorMessage={meta.error}
-                                    isInvalid={hasError}
+                                    error={hasError}
                                     label={fieldLabel}
-                                    onDateChange={(date: string | null) => {
-                                        form.setFieldValue('document_expiry', date);
-                                    }}
```

Comment:
Removing this function for now because there's no validation for the date picker Input.